### ISSUE TITLE
Compatibility List

### ISSUE THREAD

**Issue by Batwam** [Moon-0xff/gnome-mpris-label#97]
For information, list of players tested and known incompatibilities below. Before reporting an issue please check in case this is a know issue. 

If you are having issues with any of the players listed as compatible or have tested players not listed below, feel free to comment and I will update the list accordingly.

Player Name| Song Info | Album Cover | Mouse Controls | App Volume | Know incompatibilities
   :---               |    :----:         |    :----:              |    :----:                   |    :----:              |    :----      
Amberol       |         ✓        |         ✓             |         ✓                  |         ✓             |  
Brave            |         ✓        |         ✓             |         ✓                  |        ✗                | Identifies as `chromium` ¹ 
Chrome        |         ✓        |         ✓             |         ✓                  |        ✗                | Identifies as `chromium` ¹ 
Chromium   |         ✓        |         ✓             |         ✓                  |        ✗                | Identifies as `chromium` ¹ 
Firefox          |         ✓        |         ✓             |         ✓                  |         ✓              |
G4Music       |         ✓        |         ✓             |         ✓                  |         ✓              |
Monophony |         ✓        |         ✓             |         ✓                  |         ✓              |
Rhythmbox  |         ✓        |         ✓             |         ✓                  |         ✓              |
Shortwave   |         ✓        |         ✓              |         ✓                  |         ✓              |
Spotify          |         ✓        |         ✓              |         ✓                  |         ✓              | Note 2
Spotube       |         ✓        |         ✓              |         ✓                  |         ✓              | Note 2
TIDAL Hi-Fi   |         ✓        |         ✓              |         ✓                  |         ✗              | Identifies as `chromium` ¹ (#95)
Totem           |         ✓        |         ✓              |         ✓                  |         ✓              |
Vivaldi          |         ✓        |         ✓              |         ✓                  |        ✗                | Identifies as `chromium` ¹ 
VLC               |         ✓        |         ✓              |         ✓                  |         ✓              | Note 2

1: Electron apps identify as `chromium` in the Gnome Volume Mixer and the app volume therefore cannot reliably be controlled as it is not possible to differentiate between multiple `chromium` apps.
2: Might not provide metadata when started and require song skip to start showing data (#42)

**Comment by superjqp** [comment]
mpv is incompatible, by the way

**Comment by Batwam** [comment]
@superjqp in what way? Is it because it doesn't implement mpris? I did a quick google search and it looks like there is a project providing an mpris plugin for mpv

**Comment by superjqp** [comment]
uh, I only know when I play media in mpv it doesn't show in the gnome shell top bar. I'll look into this plugin, thanks.

**Comment by Batwam** [comment]
I can confirm that the extension is fully functional with mpv as long as mpv-mpris is installed. covers, controls,volume control, window focus... everything works as intended: `sudo apt install mpv-mpris` or equivalent for your favourite distro.

**Comment by Batwam** [comment]
@superjqp you can try this branch where I implemented mpris volume controls from the extension: https://github.com/Batwam/gnome-mpris-label-batwam/tree/mpris_volume_control

**Comment by Batwam** [comment]
@7system7 I haven't tried this specific app but we worked on a similar issue recently as part of of #133. Please see if the latest version in `main` fixes this for you. It should work if the app you are using implements Mpris Raise. If it doesn't, you might want to ask them to implement it as it's part of the standard.

**Comment by 7system7** [comment]
Thank you for your answer. Since then, I switched to [this](https://github.com/Andbal23/dynamic-music-pill) extension. 🙂

**Comment by 7system7** [comment]
Btw, sadly the High Tide does not implement the MPRIS completely.... and it seems, they will not in the near future. I switched to [this](https://flathub.org/apps/com.mastermindzh.tidal-hifi) player back. (Meanwhile there is a lot of GTK4 based Tidal player, but still none of them is better than this I mentioned.)

**Comment by Batwam** [comment]
yeah, that's sometimes the problem. I do report issues when I find that the protocol isn't implemented properly/fully and most devs are receptive but it's not always on high on their priorities (especially proprietary apps like plexamp/spotify...). For info, I have checked Amberol which also closes as 'Background App" (which I hate) and the new implementation is able to reopen the window using Mpris `Raise()`

**Comment by 7system7** [comment]
Currently, we have 5 usable Tidal client.
 - The first one's problem it is Electron (I think, I say everything with this.) sometimes lost the whole login session. So you randomly have to log in again.
 - we have a [Tonearm](https://flathub.org/apps/dev.dergs.Tonearm) - it crashes after every resume from suspend (it is 403 error under the hood)
 - [High Tide](https://flathub.org/apps/io.github.nokse22.high-tide) did not implement the MPRIS
 - [SONE](https://flathub.org/apps/io.github.lullabyX.sone) has a lot of issues, it is quite new
 - [hiresTI](https://github.com/yelanxin/hiresTI) is new too.

Actually I reported all issues in all repo, except the High Tide. 🙂 But after I while, it is too much. 2 weeks ago I said I will wait until a full-featured player comes. 😆 Until then, the "original" [TIDAL Hi-Fi](https://flathub.org/apps/com.mastermindzh.tidal-hifi) do that, it has to do.

***

**It is very interesting thing in open-source: a lot of developer feels strong enough to start a whole new project, but none of them thinking about to complete only one project to be perfect through contribution.**
